src/clients/UBAClient/UBAClientUtilities.ts

```diff
@@ -109,35 +117,19 @@ function omitDefaultKeys<T>(obj: Record<string, T>): Record<string, T> {
 }
 
 /**
- * Returns the UBA config for a given chainId and tokenSymbol at a given block height.
- * @param hubPoolClient The hub pool client to use for fetching the UBA config. This client's config store client must be updated.
- * @param chainId The chainId to fetch the UBA config for.
- * @param tokenSymbol The token symbol to fetch the UBA config for.
- * @param blockNumber The block height to fetch the UBA config for.
- * @returns The UBA config for the given chainId and tokenSymbol at the given block height.
- * @throws If the config store client has not been updated at least once.
- * @throws If the L1 token address cannot be found for the given token symbol.
- * @throws If the UBA config for the given block height cannot be found.
+ * Parses a UBAParsedConfigType into a UBAFeeConfig
+ * @param chainId The chain ID to parse the UBA config for
+ * @param tokenSymbol The token symbol to parse the UBA config for
+ * @param ubaConfig The UBAParsedConfigType to parse
+ * @returns The parsed UBAFeeConfig
  */
-export function getUBAFeeConfig(
```

Comment:
What exactly did you change here?

Reply:
The diff is just being weird here. The definition of `getUBAFeeConfig` begins on line 197.

The change i've made here was to separate the parsing of the UBAConfig out of the `getUBAFeeConfig` so that I can use it in the relayer-v2 unit tests. Nothing functionally changes here except that the logic for parsing the UBA config is now in `parseUBAFeeConfig`. It's called on line 214 of this file.

Reply:
ok cool